pkg/console/validator.go

```diff
@@ -201,6 +202,11 @@ func checkNetworks(networks map[string]config.Network) error {
 		default:
 			return prettyError(ErrMsgNetworkMethodUnknown, network.Method)
 		}
```

Comment:
VlanID > 4094, is error

https://en.wikipedia.org/wiki/IEEE_802.1Q
VLAN identifier (VID)
A 12-bit field specifying the VLAN to which the frame belongs. The values of 0 and 4095 (0x000 and 0xFFF in hexadecimal) are reserved. All other values may be used as VLAN identifiers, allowing up to 4,094 VLANs. The reserved value 0x000 indicates that the frame does not carry a VLAN ID

Reply:
Updated it. Thanks :)